### ISSUE TITLE
@onflow/cadence-parser Cannot use import statement outside a module

### ISSUE THREAD

**Issue by LemonNekoGH** [onflow/cadence#1873]
### Problem
I can't use `@onflow/cadence-parser` to parse my Cadence code, because `Cannot use import statement outside a module` error.

### Steps to Reproduce
1. pnpm i @onflow/cadence-parser
2. write code below
    ```
    import {CadenceParser} from "@onflow/cadence-parser"
    const parser = await CadenceParser.create("cadence-parser.wasm")
    ```
3. run your project
4. see the error
    ```
    /Users/lemonneko/transform/node_modules/@onflow/cadence-parser/dist/index.js:27
    import { go } from './go.js';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
    ```

**Comment by gregsantos** [comment]
Hi @LemonNekoGH 
Mind providing a bit more context here, specifically on 3. Run your project.
Are you running this with node, if so what version?

**Comment by LemonNekoGH** [comment]
run program using node 16 with pnpm

**Comment by LemonNekoGH** [comment]
i tryed to build package from your latest code, errors disappeared. the latest code seems sloved this promblem.

**Comment by LemonNekoGH** [comment]
![image](https://user-images.githubusercontent.com/17664845/185735770-055956fe-e7c6-4055-8613-3b44a747d05c.png)
this package with version 0.20.0 is published 9 mounth ago and latest tag is 0.25.0

**Comment by turbolent** [comment]
This should be fixed now. Please open a new issue if there is still a problem
